File: php/laravel/app/Models/BankAccount.php
<?php

namespace App\Models;

use App\Enums\Banks;
use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class BankAccount extends Model
{
    use HasFactory;

    protected $fillable = [
        'user_id',
        'number',
        'agency',
        'bank',
        'state',
        'city',
        'balance',
    ];

    protected $casts = [
        'bank' => Banks::class
    ];
}

File: api/go/gin/app/models/bank_account.go
package models

import "time"

type BankAccount struct {
	ID        uint64    `json:"id"`
	UserId    uint64    `json:"user_id"`
	Number    string    `json:"number"`
	Balance   int64     `json:"balance"`
	CreatedAt time.Time `json:"created_at"`
	UpdatedAt time.Time `json:"updated_at"`
}
